modules/giapi/src/main/scala/giapi/client/package.scala

```diff
@@ -156,18 +163,29 @@ package client {
       * @param url Url to connect to
       * @tparam F Effect type
       */
+    // scalastyle:off
```

Comment:
This is a case where the IMHO the line limit check in scala style is broken. The method includes the creation of a trait instance which is long but composed of small methods